test/test_isl.py

```diff
@@ -71,6 +71,22 @@ def test_subst_into_pwqpolynomial():
     assert (result - expected_pwqpoly).is_zero()
 
 
+def test_subst_into_pwaff():
+    from pymbolic.primitives import Variable
+    arg_dict = {
+            "m": 3*Variable("nx"),
+            "n": 2*Variable("ny")}
```

Comment:
Maybe also test an offset to make sure that works, too?

Reply:
Yep. 97a754f3.